File: Java/Scuola/src/Corso.java
public class Corso {
    
    // Assigning Variables
    private String nomeCorso;
    private String matricola;
    private int durataAnni;
    
    // Constructor
    public Corso(String nomeCorsoString, String matricolaString, int durataAnniInt) {
        this.nomeCorso = nomeCorsoString;
        this.matricola = matricolaString;
        this.durataAnni = durataAnniInt;
    }

    // Getters
    public String getNome() {
        return this.nomeCorso;
    }

    public String getMatricola() {
        return this.matricola;
    }

    public int getDurata() {
        return this.durataAnni;
    }

    // Setters
    public void setNome(String nomeCorsoString) {
        this.nomeCorso = nomeCorsoString;
    }

}

File: Py/Classi/verifica tryhard/corso.py
class Corso:
    
    def __init__(self, nome, matricola, durata):
        self.nome = nome
        self.matricola = matricola
        self.durata = durata

    def getNome(self):
        return self.nome
    
    def getMatricola(self):
        return self.matricola
    
    def getDurata(self):
        return self.durata
    
    def setNome(self, nome):
        self.nome = nome

    def setMatricola(self, matricola):
        self.matricola = matricola

    def setDurata(self, durata):
        self.durata = durata